com.pot_app.pot.yml

```diff
@@ -0,0 +1,154 @@
+id: com.pot_app.pot
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: pot
+rename-icon: pot
+rename-desktop-file: pot.desktop
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  - name: webkit2gtk-4.0
```

Comment:
why can't you use the version in the runtime instead?

Reply:
webkitgtk has no effect on tray...